### PR TITLE
Add product to order jl

### PR DESCRIPTION
**Pull request by joshualloyd**
Related Issue: #5 

Description:
User can add products to an order. This will create an order if an open one doesn't exist for the customer, otherwise it will add the product to the current open order(or the first open one listed until we fix data generation to only have one open order per customer.

Notes for Readme Info:
At the main menu first select a customer (option 2), then after selecting a customer select 'add product to shopping cart' where you will be given a list of the products. Select one (1-5), then determine a quantity (any number). The order line item will be updated and you will go back to the main menu.

Chai/Mocha Tests added:
test/add-to-order.test.js

Any other PR Testing Steps:
Run the tests with 'npm test'
Also walk through the menu to test the UI.

### CODE REVIEW

File: app/controllers/add-to-order-ctrl.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    dbGetAllProducts()
+      .then((prodArr) => {
+        for (let i = 0; i < prodArr.length; i++) {
+          console.log(`${prodArr[i].id}: ${prodArr[i].title}, ${prodArr[i].description}`);
+        }
+        prompt.get(
+          [
+            {
+              name: 'product',
+              description: 'Choose a product to add to the order?'
```
Comment (thejonroberts) [inline app/controllers/add-to-order-ctrl.js]: 'Choose a product ID to add to the order'

Comment (thejonroberts) [reply app/controllers/add-to-order-ctrl.js]: make a TODO to refactor this 50 line function out later

File: package.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "db:reset": "node bin/db-reset",
     "start": "node bin/bangazon",
-    "test": "mocha -R list --watch"
+    "test": "mocha --watch --recursive -R spec"
```
Comment (thejonroberts) [inline package.json]: this is dope!

File: app/ui.js [hunk 3]
```diff
@@ -6,6 +6,9 @@ const { red, magenta, blue } = require('chalk');
 const prompt = require('prompt');
 const colors = require('colors/safe');
 prompt.message = colors.blue('Bangazon Corp');
+const { promptPrintUsers } = require('./controllers/active-user-ctrl');
```
Comment (thejonroberts) [inline app/ui.js]: these were defined above, throwing errors

Comment (joshualloyd) [reply app/ui.js]: fixed

**Comment by joshualloyd** [comment]
Updated the PR branch

**Pull request merged**